### PR TITLE
New package: RAGEMultiplayerTeam.RAGEMultiplayer version 1.1.0.0

### PR DESCRIPTION
**Pull request by rakleed**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135612)

### CODE REVIEW

**Comment by rakleed** [comment]
https://rage.mp/

**Comment by rakleed** [comment]
What?

**Comment by stephengillie** [comment]
This might be due to IP blocks of data centers.

File: manifests/r/RAGEMultiplayerTeam/RAGEMultiplayer/1.1.0.0/RAGEMultiplayerTeam.RAGEMultiplayer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: RAGEMultiplayerTeam.RAGEMultiplayer
+PackageVersion: 1.1.0.0
+PackageLocale: en-US
+Publisher: RAGE Multiplayer Team
+PackageName: RAGE Multiplayer
+PackageUrl: https://rage.mp/
```
Comment (stephengillie) [inline manifests/r/RAGEMultiplayerTeam/RAGEMultiplayer/1.1.0.0/RAGEMultiplayerTeam.RAGEMultiplayer.locale.en-US.yaml:9]:
```suggestion
# PackageUrl: 
```

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

**Comment by stephengillie** [comment]
Hi @rakleed,

It seems like GTA V from Rockstar might be a dependency. Unfortunately, we don't currently have `Rockstar.GTA5` among our manifests. 

Thoughts?

**Comment by rakleed** [comment]
Indeed, without the GTA 5 game this app is useless.

But GTA 5 is a proprietary paid game that can only be downloaded by purchasing it first, so it cannot be added to winget. Then there is no point in adding the RAGE Multiplayer app to the winget, right?

**Review by stephengillie — APPROVED** [review]

**Pull request merged**